client/src/components/Artist.jsx

```diff
@@ -50,13 +50,13 @@ class Artist extends Component {
   }
 
   componentWillMount() {
-    //should have a detailed, user customized profile
+    // should have a detailed, user customized profile
     let { dispatch } = this.props;
     let artistId = this.props.match.params.artistId;
     dispatch(ArtistAction.fetchingArtist(true));
-    fetch('/artist/' + this.props.match.params.artistId)
+    fetch(`/artist/${this.props.match.params.artistId}`)
```

Comment:
I know that they're essentially the same thing, but why does it matter if its es5 vs es6?